test/core/symbol/SymbolNetwork.spec.ts

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright 2021 SYMBOL
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { SymbolNetwork } from '@core';
+import { expect } from 'chai';
+import { keccak256, sha3_256 } from 'js-sha3';
+import { BasicNetworkTester } from '../../BasicNetworkTest.template';
+
+describe('Symbol Network', () => {
+    // Arrange:
+    const networks = [
+        new SymbolNetwork('public', 0x68, '57F7DA205008026C776CB6AED843393F04CD458E0AA2D9F1D5F31A402072B2D6'),
+        new SymbolNetwork('public_test', 0x98, '3B5E1FA6445653C971A50687E75E6D09FB30481055E3990C84B25E9222DC1155'),
```

Comment:
i think @0x6861746366574 wanted to go away from public/public_test and towards mainnet/testnet?